migrations/sql/V2019.03.12_1523__InitialSchema.sql

```diff
@@ -26,10 +26,9 @@ PRIMARY KEY ("id")
 );
 
 CREATE TABLE "orderlyweb_user_group_permission" (
-"id" SERIAL ,
+"id" INTEGER PRIMARY KEY NOT NULL,
```

Comment:
Jooq was trying to insert null ids. After changing the sql syntax here and regenerating the interface it is now working as expected.

Reply:
Yeah, looks like sqlite doesn't support SERIAL, but does have AUTOINCREMENT. Do we want this to autoincrement? I guess we probably do, if we can trust sqlite to do it nicely!

Reply:
Yeah I looked into this and sqlite documentation recommends not to use `autoincrement`. It will auto increment integer primary keys by default, adding the `autoincrement` keyword just enforces that old ids (of deleted rows) are not re-used: http://www.sqlitetutorial.net/sqlite-autoincrement/